src/Discovery/DiscoveryActionBar/components/OpenFillRequestFormButton.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```

Comment:
this whole file need some linting, consider running eslint over this file

Reply:
Done